firefox-ios/Client/Frontend/Library/Reader/ReaderPanelEmptyStateView.swift

```diff
@@ -0,0 +1,167 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+class ReaderPanelEmptyStateView: UIView {
+    let windowUUID: WindowUUID
+    let themeManager: Common.ThemeManager
+
+    private lazy var welcomeLabel: UILabel = {
```

Comment:
nit: to keep consistent with how we initialize other UI elements
```
private lazy var welcomeLabel: UILabel = .build {
```

Reply:
Sure! Thanks for pointing that out, I'll make the change.